arangod/Cluster/ActionBase.h

```diff
@@ -30,7 +30,10 @@
 #include "Basics/Common.h"
 #include "Basics/Result.h"
 
+#include <atomic>
```

Comment:
Are those includes needed here?

Reply:
(also the other ones below which have been added)

Reply:
These includes may not be 100% necessary here (in fact, they were not present before). But this is only because previous file includes will have incldued `atomic`, `memory` etc., so we relied on other includes here to include the right headers for us.
IMHO this is not how it should be. We should follow the include-what-you-use rule, and Action.h base uses `std::shared_ptr` (memory) and `std::atomic` (atomic), std::chrono (chrono) etc. 
So yes, we should include these headers here!

Reply:
Ty!